src/bot/mtproto-api/workrooms.ts

```diff
@@ -0,0 +1,197 @@
+import { Context, SupportedEvents } from "#root/types/context";
```

Comment:
Import directory syntax is interesting #root

Reply:
It came with the template and I kept it for my own corner cutting but I can refactor/replace if necessary. I assumed it had something to do with the build output of the template re: import paths but we rarely have those sorts of issues in other plugins so likely not required.